proposed/simplecache.md

```diff
@@ -114,85 +112,117 @@ namespace Psr\SimpleCache;
 interface CacheInterface
 {
     /**
-     * Fetch a value from the cache.
+     * Fetches a value from the cache.
+     *
+     * @param string $key     The unique key of this item in the cache.
+     * @param mixed  $default Default value to return if the key does not exist.
      *
-     * @param string $key     The unique key of this item in the cache
-     * @param mixed  $default Default value to return if the key does not exist
+     * @return mixed The value of the item from the cache, or $default in case of cache miss.
      *
-     * @return mixed The value of the item from the cache, or $default in case of cache miss
+     * @throws InvalidArgumentException
+     *   If the $key string is not a legal value a \Psr\SimpleCache\InvalidArgumentException
```

Comment:
updated. I kept the normative MUST which is important to me to force an imperative interpretation (that's why RFC 2119 is for)